persistence/src/main/java/cz/muni/fi/pa165/mushrooms/entity/Visit.java

```diff
@@ -37,6 +31,10 @@
     @JoinColumn(name = "visit_forest", nullable = false)
     private Forest forest;
 
+    @ManyToMany

```

Comment:
I think this should be @OneToMany, because we don't want to store Visits in Mushroom

Reply:
resolved